### ISSUE TITLE
Some uploaded videos do not play (their embeds are non-interactable)

### ISSUE THREAD

**Issue by Wendell250** [vector-im/element-web#21809]
### Steps to reproduce

1. Upload a video (drag&drop or via paperclip)
2. Video uploads like usual
3. Video is unplayable

#### What did you expect?
Video to be playable.

#### What happened instead?
The opposite.

After the recent update (1.10.9) *some* video uploads turn out non-interactable.

The upload process works fine, including "Upload files" preview.
![vid_issue1](https://user-images.githubusercontent.com/90041836/163474641-267a2e0a-7b9e-4163-99dc-4ab52f514344.png)

But after that the embed is greyed-out and you can't click "play".
![vid_issue2](https://user-images.githubusercontent.com/90041836/163475159-26f10554-6ed9-46c3-bd4c-7bd38dc3b004.png)

This issue seems to affect videos at random.
Some videos that 100% worked before now become unplayable.

Here are three examples:
1. [broken1](https://user-images.githubusercontent.com/90041836/163475521-4cbefcfa-b462-4920-a453-b9face3d2232.mp4)
2. [broken2](https://user-images.githubusercontent.com/90041836/163475571-8c31893c-5df9-4892-a375-8bc2a387ba9d.mp4)
3. [broken3](https://user-images.githubusercontent.com/90041836/163475581-9366239e-95d3-48fb-9906-b80a272cd2ff.mp4)

All of them play fine in the browser (FF).

`broken2` doesn't open in VLC, giving the following error:
> No suitable decoder module:
> VLC does not support the audio or video format "av01". Unfortunately there is no way for you to fix this.

And just in case [one example](https://user-images.githubusercontent.com/90041836/163476023-f21e2be2-47b4-4bed-89bd-c955529f980c.mp4) of a working one.

***

Specs:

Threads are enabled.
I'm on Windows 8.1, but my buddies on Windows 10 are also experiencing this.

### Operating system

Windows 8.1

### Application version

Element version: 1.10.9; Olm version: 3.2.8

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by konomikitten** [comment]
This is affecting me on Linux.

Operating system: Debian Unstable
Application version: Element version: 1.10.9 Olm version: 3.2.8
Homeserver: matrix.org

**Comment by t3chguy** [comment]
All 3 of your sample videos play fine for me in FF on MacOS

![image](https://user-images.githubusercontent.com/2403652/164033534-482c6d65-9300-4b54-9803-42da7cfa89e2.png)

**Comment by Wendell250** [comment]
> All 3 of your sample videos play fine for me in FF on MacOS

Yes, I mentioned that as well:
> All of them play fine in the browser (FF).

But I submitted the issue for Element Desktop (1.10.9).
Guess the Github form doesn't append that, so my bad for not specifying it.

I'm now on `1.10.10` and the issue is still here:

![video_issue](https://user-images.githubusercontent.com/90041836/164071865-83dba78c-69fa-4c6a-b074-344e916fe1b8.png)

**Comment by Wendell250** [comment]
> Filling the template helps

Yeah, my bad.

> Finder also has an issue with broken2

Same for Windows Explorer.

> All 3 videos work for me in Element Desktop on MacOS so I still cannot reproduce

Two of my friends on Windows 10, and one on MacOS (old macbook, though), have the same issue with all 3 videos.
All on Element Desktop (1.10.9/1.10.10).

I can try to ask them to submit rageshakes as well, if that's gonna be helpful.

**Comment by talizea** [comment]
If you send the videos in a telegram chat (they don't re-encode; exact same file) and bridge it to matrix, you can play the video in element just fine. Bug only occurs on the uploaders end when used in element directly. Win 11, element desktop.

> All 3 videos work for me in Element Desktop on MacOS so I still cannot reproduce

Wild idea: Have you considered to try this on a platform where the bug is reported to exist? 😉

**Comment by t3chguy** [comment]
> Wild idea: Have you considered to try this on a platform where the bug is reported to exist? 

If it is a bug only in one OS for the same browser then chances are the bug is an upstream one with the browser given the browser is meant to be acting as a consistent abstraction layer regardless of the underlying OS. I don't have a Windows computer and certainly not a Windows 8.1 one.

**Comment by talizea** [comment]
Well, it occurs in Windows 8, 10, 11 and Linux apparently.

**Comment by Wendell250** [comment]
Interesting.
I decided to make a new encrypted and an unencrypted room.
It still fails in the encrypted one, but it seems to work just fine in the unencrypted room.

I've submitted the rageshake again.

![enc](https://user-images.githubusercontent.com/90041836/172938791-8bd55cb0-05f0-4bfa-bf50-7af3e0defe16.png)
![unenc](https://user-images.githubusercontent.com/90041836/172938818-3e019329-d920-43b7-86b1-00c7e8eba47f.png)
